enclave-runtime/Cargo.lock

```diff
@@ -15,7 +15,7 @@ dependencies = [
 [[package]]
 name = "ac-compose-macros"
 version = "0.1.0"
-source = "git+https://github.com/scs/substrate-api-client?branch=master#a4d931c85f30ba22ff1f982c47caf57fe43ebc90"
+source = "git+https://github.com/scs/substrate-api-client?branch=master#1a5951450203b11cb347d624891ba608e2e27f6d"
```

Comment:
Is this a newer or older revision than in PR #556 by @gaudenzkessler (https://github.com/integritee-network/worker/pull/556/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87e)? We should probably put in the same revision in both PRs to prevent problems when rebasing (and potentially 'reverting' back to the wrong version)

Reply:
#556 requires minimum commit 87595eb your version (1a59514) is newer so its fine!

Reply:
Okay, so we just have to keep that in mind when we rebase (whichever PR is merged later)